packages/cli/src/run.js

```diff
@@ -7,7 +7,7 @@ import setup from './setup'
 import getCommand from './commands'
 
 function checkDuplicateNuxt() {
-  const dupPkg = pkgName === '@nuxt/cli' ? 'cli-edge' : 'cli'
```

Comment:
`@nuxt/cli` will be replaced to `@nuxt/cli-edge` by rollup, so `pkgName === '@nuxt/cli'` is always true